### PR TITLE
Update /company-culture copy

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Updated copy as per [doc](https://docs.google.com/document/d/1PAeP_ZO4JzSBCi-fRp6F7SYtSSRY-4DcWLUf9HJaMxs/edit)

## QA

- View the site locally in your web browser at: https://canonical-com-991.demos.haus/careers/company-culture
- See that copy changes were made
- Check different screen sizes to ensure there are no orphans in the text 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4811

### CODE REVIEW

File: templates/careers/company-culture/index.html [hunk 1]
```diff
@@ -87,8 +87,7 @@ <h3 class="p-muted-heading">Built on trust</h3>
     </div>
     <div class="col-medium-4 col-9">
       <div class="p-strip is-shallow u-no-padding--top">
-        <p>As a remote-first organisation, trust is our currency. At Canonical, you get unprecedented flexibility in how you organise your day and freedom to explore new roles within the company.
-          We trust each other to be organised and effective in a distributed workplace. If you value teamwork and are committed to being a good counterpart to colleagues around the world, you will love working here.</p>
+        <p>As a remote-first organisation, trust is our currency. We hire and invest in people who are organised and effective. If you value teamwork and are committed to being a good counterpart to colleagues around the world, you will love working here.</p>
```
Comment (carkod) [inline templates/careers/company-culture/index.html:90]: Missing "self-driven". "organised, self-driven and effective"

File: templates/careers/company-culture/index.html [hunk 2]
```diff
@@ -125,7 +124,7 @@ <h3 class="p-muted-heading">Committed <br class="u-hide--small">
     </div>
     <div class="col-medium-4 col-9">
       <div class="p-strip is-shallow u-no-padding--top">
-        <p>We aspire to lead on the global stage. Most of us are avid students of greatness &dash; we are interested in how the latest technologies work, how the best organisations work and how the smartest teams collaborate. We aim to hold our own in that crowd. We pursue engineering excellence and love to engineer excellence in all aspects of our work. From releasing the world’s most popular Linux reliably to developing our <a href="/documentation">documentation</a> practice.
+        <p>We aspire to lead on the global stage. Most of us are avid students of greatness &dash; we are interested in how the latest technologies work, how the best organisations work and how the smartest teams collaborate. We aim to hold our own in that crowd. We pursue engineering excellence and love to engineer excellence in all aspects of our work. From releasing the world's most popular Linux reliably to trailblazxing <a href="/documentation">documentation</a>.
```
Comment (carkod) [inline templates/careers/company-culture/index.html:127]: "trailblazing"? Not even sure if the copy doc is right :dancers:

File: templates/careers/company-culture/index.html [hunk 3]
```diff
@@ -177,7 +176,7 @@ <h3 class="p-muted-heading">Life at Canonical</h3>
           <h2><a href="/careers/company-culture/remote-work">Remote work</a></h2>
         </div>
         <div class="col-6 col-medium-3">
-          <p>We are remote but connected. The walls that exist in a co-located environment don’t exist here. Find out how we collaborate and what we value in our fully-distributed team.</p>
+          <p>We are remote but we neet each other at least twice a year during in-person sprints. Find out how we collaborate.</p>
```
Comment (carkod) [inline templates/careers/company-culture/index.html:179]: "meet" each other

**Review by carkod — APPROVED** [review]
LGTM just a few typos.

File: templates/careers/company-culture/index.html (hunk 2 above)
Comment (mtruj013) [reply templates/careers/company-culture/index.html]: I think trailblazing is correct 🙂

**Pull request merged**